source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -55,7 +55,7 @@ def make_url_invalid_code_test():
     """
     sytral = Sytral(id='tata', service_url='http://bob.com/')
 
-    url = sytral._make_url(MockRoutePoint(line_code='line_toto', stop_id=None))
+    url = sytral._make_url(MockRoutePoint(line_code='line_toto', stop_id=[]))
```

Comment:
minor: you can rename to show it's a list
```suggestion
    url = sytral._make_url(MockRoutePoint(line_code='line_toto', stop_id_list=[]))
```
or just plural like for _hardcoded_stop_id`s` in tests?